tests/tests_cluster_basic_custom_pcsd_tls.yml

```
@@ -5,11 +5,18 @@
   vars_files: vars/main.yml
   vars:
     ha_cluster_cluster_name: test-cluster
-    ha_cluster_pcsd_public_key_src: "{{ playbook_dir }}/tmp/pcsd.crt"
-    ha_cluster_pcsd_private_key_src: "{{ playbook_dir }}/tmp/pcsd.key"
+    ha_cluster_pcsd_public_key_src: "{{ __ha_cluster_work_dir.path }}/pcsd.crt"
+    ha_cluster_pcsd_private_key_src: "{{ __ha_cluster_work_dir.path }}/pcsd.key"
 
   tasks:
     - block:
+        - name: Create temp directory for secrets
+          tempfile:
+            state: directory
+            prefix: lsr_ha_cluster_
+          register: __ha_cluster_work_dir
+          delegate_to: localhost
```

```suggestion
          delegate_to: localhost
          run_once: true
```
I'm running the test on a two-node cluster. Without `run_once: true`, each node gets a different work directory path, and the test fails.